content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -84,45 +84,309 @@ If `--config` is provided and the values are not specified via the command line,
 defaults for the `KubeletConfiguration` version apply.
 In the above example, this version is `kubelet.config.k8s.io/v1beta1`.
 
+## Viewing the kubelet configuration
+
+To inspect the kubelet configuration, follow these steps:
+
+1. Start a proxy server using `kubectl proxy` in your terminal.
```

Comment:
Is this code always going to work? 

I don't know much about `kubectl proxy` but this seems that it would be locked down from most users using managed cloud services?

And do we need to pass any certs or tls for curl for this to work?

Reply:
There's another way to query the kubelet config
```bash
$ kubectl get --raw "/api/v1/nodes/<node-name>/proxy/configz" | jq
```
which doesn't involve passing any certs,

Reply:
Yeah, that's tricky. I wrote this documentation with the normal use case in mind.

Reply:
I don't know what normal would be in this case. I would usually suggest mentioning links to `kubectl proxy` or something more detailed in case it doesn't work for the user..